plugins/openshift/skills/generating-ovn-topology/scripts/collect_ovn_data.py

```
@@ -0,0 +1,654 @@
+
+        Returns:
+            (success: bool, count: int) - Whether query succeeded and entry count
+
+        Note:
+            All exceptions (timeouts, subprocess errors) are caught and handled
+            internally. Errors are logged to stderr and return (False, 0).
+        """
+        try:
+            env = os.environ.copy()
+            env["KUBECONFIG"] = self.kubeconfig
+            result = subprocess.run(
```

Ruff (0.14.4)

`subprocess` call: check for execution of untrusted input

(S603)